site/content/docs/manifest/backend-service.en.md

```diff
@@ -141,6 +141,31 @@ List of all available properties for a `'Backend Service'` manifest. To learn ab
               efs:
                 id: fs-1234567
         ```
+        === "Multiple Ports Exposed"
```

Comment:
Not sure which name is more interesting and descriptive 🤔  Throwing some ideas here

```suggestion
        === "Expose multiple ports"
```

```suggestion
        === "Multiple paths"
```